.github/workflows/api.yml

```diff
@@ -0,0 +1,106 @@
+name: api
+
+on:
+  push:
+    paths:
+      - "apigo/**"
+
+jobs:
+  lint:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - run: |
+          echo "noop"
```

Comment:
Why keeping this job?

Reply:
I thgink it's ok to keep it as a placeholder for later (which is soon)

Reply:
Lint should be enforced for go (ci and precommit). Eventually this job must be filled with the linting strategy well agree upon. Keeping the spot to make the lint integration a no-brainer.